lib/crystalball/source_diff.rb

```diff
@@ -28,7 +28,18 @@ def empty?
 
     # TODO: Include untracked to changeset
     def changeset
-      @changeset ||= git_diff.map { |file_diff| FileDiff.new(file_diff) }
+      @changeset ||= git_diff.map do |file_diff|
+        diff = FileDiff.new(file_diff)
+        # FIXME: ruby-git marks moved files as modified. It works for lib files, but not for specs
```

Comment:
I remember testing it and it was the opposite. `ruby-git` instead of marking files as moved, would mark them as old file removed, new file added.

Reply:
hm. a difference of versions maybe? I would like to test it if possible. On my machine, it is `modified`